SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosTaskBuilder.java

```diff
@@ -184,6 +184,8 @@ private void prepareEnvironment(final SingularityTaskRequest task, SingularityTa
     setEnv(envBldr, "TASK_ID", taskId.getId());
     setEnv(envBldr, "ESTIMATED_INSTANCE_COUNT", task.getRequest().getInstancesSafe());
 
+    setEnv(envBldr, "JOB_USER", task.getPendingTask().getUser().or(""));
```

Comment:
Two things.
- Wondering if `JOB_USER` is descriptive enough here. This method will get called for services and workers as well. Maybe something more like `STARTED_BY_USER` ?
- In the case where `user` is not present on the pending task, would it make more sent to leave this out rather than have an empty string as the value?